glyphs-reader/src/font.rs

```diff
@@ -1471,6 +1473,9 @@ impl TryFrom<RawFont> for Font {
         let glyph_order = parse_glyph_order(&from);
         let glyph_to_codepoints = parse_codepoints(&mut from, radix);
 
+        let use_typo_metrics =
+            custom_param_int(&from.other_stuff, "Use Typo Metrics").map(|v| v == 1);
```

Comment:
you may also want to support "Has WWS Names" custom parameter (flag 8 of fsSelection) since you're touching this.
It doesn't seem like Glyphs.app allow one to set arbitrary fsSelection flags like UFO fontinfo.plist does, the style-linking ones are somewhat implied by the master name and then there's "Use Typo Metrics" and "Has WWS Names", the rest of the fsSelection flags can't be turned on apparently (or at least I haven't found how.. I suspect nobody cares about those).

Reply:
Added.